### PR TITLE
User story52

### PR DESCRIPTION
**Pull request by KellyIB**
Basically all user story 52 needed was a test.  All functionality was already written in user stories 38-41

### CODE REVIEW

File: spec/features/admin/merchant_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        expect(page).to have_button("Disable")
+      end
+
+      within "#merchant-#{@merchant3.id}" do
+        expect(page).to have_button("Enable")
+      end
+    end
+  end
+end
+
```
Comment (cgaddis36) [inline spec/features/admin/merchant_index_spec.rb:71]: I like it, tests entire story

File: spec/features/admin/merchant_index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      expect(current_path).to eq("/admin/merchants/#{@merchant3.id}")
+    end
+
+    it "can see a disable button for enabled merchants and enable button for disabled merchants" do
+      visit '/admin/merchants'
+
+      within "#merchant-#{@merchant1.id}" do
+        expect(page).to have_button("Disable")
+      end
```
Comment (adumortier) [inline spec/features/admin/merchant_index_spec.rb:59]: the functionality that `admin` can disable `merchant` by clicking the `disable` button is tested in `/spec/features/admin/admin_authorization_spec.rb`

File: spec/features/admin/users_spec.rb [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+    expect(current_path).to eq("/admin/users/#{user3.id}")
+
+    visit "/admin/users"
+
+    within "#user-#{user4.id}" do
+      expect(page).to have_link(user4.name)
+      expect(page).to have_content(user4.role)
+      expect(page).to have_content(user4.created_at)
+      click_on "#{user4.name}"
```
Comment (adumortier) [inline spec/features/admin/users_spec.rb:68]: I think we can DRY up the code here by just clicking the `user name` without checking that the link is there

File: spec/features/admin/users_spec.rb [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+                                state: 'TN',
+                                zip_code: 12345,
+                                email: "roman@example.com",
+                                password: "hamburger01",
+                                role: 1)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(johnny)
+
+    visit "/admin/users"
+
+    expect(page).to have_content("The page you were looking for doesn't exist (404)")
```
Comment (adumortier) [inline spec/features/admin/users_spec.rb:101]: cool! I found the test I was looking for :)

**Review by adumortier — APPROVED** [review]

File: spec/features/admin/users_spec.rb [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    end
+
+    expect(current_path).to eq("/admin/users/#{user3.id}")
+
+    visit "/admin/users"
+
+    within "#user-#{user4.id}" do
+      expect(page).to have_link(user4.name)
+      expect(page).to have_content(user4.role)
+      expect(page).to have_content(user4.created_at)
```
Comment (cgaddis36) [inline spec/features/admin/users_spec.rb:67]: nice job testing each specific user her, very robust

**Review by cgaddis36 — APPROVED** [review]
Everything looks good to me; thank you for adding these tests in

**Pull request merged**